subsys/testsuite/ztest/include/zephyr/ztest_assert.h

```diff
@@ -143,6 +143,12 @@ static inline bool z_zassume(bool cond, const char *default_msg, const char *fil
 		}                                                                                  \
 	} while (0)
 
+#define _zassert_va(cond, default_msg, msg, ...)                                                   \
+	_zassert_base(cond, default_msg, msg, ##__VA_ARGS__)
+
+#define zassert(cond, default_msg, ...)                                                            \
+	_zassert_va(cond, default_msg, COND_CODE_1(__VA_OPT__(1), (__VA_ARGS__), (NULL)))
```

Comment:
Is `__VA_OPT__` supported by all the toolchains supported by Zephyr? /cc @stephanosio

Reply:
For one, `__VA_OPT__` is non-standard and a GNU extension.

I am not sure if it is supported by the XCC, which is based on GCC 4.x. Can you confirm? @nashif @dcpleung @laurenmurphyx64

Reply:
looks like it is.